src/components/ContractCardForContractor/index.jsx

```diff
@@ -35,6 +37,7 @@ const {
 } = publicRuntimeConfig
 
 export const ContractCardForContractor = ({
+  jobAddress,
```

Comment:
You don't need `jobAddress` here, because `contract` already has `jobAddress` field :-)